scripts/GitHubMergeBranches.ps1

```diff
@@ -272,8 +268,29 @@ try {
         | select -First 1
 
     if ($matchingPr) {
+        $prUpdatedSuccess = $false
+
+        try {
+            if ($PSCmdlet.ShouldProcess("Update remote branch $mergeBranchName on $remoteName")) {
+                Invoke-Block { & git push $remoteName "${mergeBranchName}:${mergeBranchName}" }
+            }
+            $prUpdatedSuccess = $true
+        }
+        catch {
+            Write-Warning "Failed to update existing PR"
+        }
+
+        $prMessage = if ($prUpdatedSuccess) {
+            "This pull request has been updated.`n`n$committersList"
+        } else {
+            @"
+:x: Uh oh, this pull request could not be updated automatically. New commits were pushed to $HeadBranch, but I could not automatically push those to $mergeBranchName to update this PR.
```

Comment:
Clearly our bot should speak in GIFs :)

![https://media1.tenor.com/images/b3a35e03d8206b7b960cd69cc17b2c96/tenor.gif?itemid=5708794](https://media1.tenor.com/images/b3a35e03d8206b7b960cd69cc17b2c96/tenor.gif?itemid=5708794)

Reply:
Haha